TextureExtraction tool/Data/TextureExtractor.cs

```diff
@@ -50,22 +51,27 @@ public class ExtractorOptions : Options
             public TextureActionDelegate TextureAction;
 
             public delegate void TextureActionDelegate(JUTTexture.TexEntry texture, Results results, in string subdirectory);
+
+            public ExtractorOptions()
+            {
+                if (!UseConfig) return;
+
+                if (bool.TryParse(Config.Get("Mips"), out bool value)) Mips = value;
+                if (bool.TryParse(Config.Get("Raw"), out value)) Raw = value;
+                if (bool.TryParse(Config.Get("Force"), out value)) Force = value;
+                if (bool.TryParse(Config.Get("DolphinMipDetection"), out value)) DolphinMipDetection = value;
+                if (bool.TryParse(Config.Get("Cleanup"), out value)) Cleanup = value;
+                if (int.TryParse(Config.Get("Tasks"), out int thing)) Parallel.MaxDegreeOfParallelism = thing;
+            }
         }
 
         public class ExtractorResult : Results
         {
             public TimeSpan TotalTime { get; internal set; }
 
-            public int MinExtractionRate => (int)Math.Round((double)100 / (ExtractedSize + SkippedSize + UnsupportedSize) * ExtractedSize);
+            public int MinExtractionRate => MathEx.RoundToInt(100d / (ExtractedSize + SkippedSize + UnsupportedSize) * ExtractedSize);
```

Comment:
I think MathEx.RoundToInt is a bit redundant, but it doesn't really bother me.

Reply:
Unity brainrot made me want parity on their syntax, because I certainly acknowledge this as the most redundant introduction. Maybe a Round function with IConvertible generic type input/output would be the better route for something in MathEx, but I'm not gonna bother with that for now